queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py

```diff
@@ -109,6 +109,9 @@ def correct_business_data(business: Business,  # pylint: disable=too-many-locals
         resolution = dpath.util.get(correction_filing, '/correction/resolution')
         filings.update_filing_json(correction_filing_rec, resolution)
         resolutions.update_resolution(business, resolution)
+        if resolution:
+            filing_meta.correction = {**filing_meta.correction,
```

Comment:
Hiding this if there are no resolution changes. Only documents that show change hopefully.